### PR TITLE
feat: Add `silent` log level for Cloud Code

### LINKED ISSUE #8418: Avoid stringify if log levels are not met
### New Issue Checklist
- [x ] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [ x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [ x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description
Sorry I don't have the time for a PR as I struggle with my day-to-day job, but as I told @mtrezza I think we can have a significant performance increase by rewriting the [`logTriggerSuccessBeforeHook`](https://github.com/parse-community/parse-server/blob/8414ad3f3282bc0bb3b43e55396998f4f9a7444d/src/triggers.js#L390) and [`logTriggerErrorBeforeHook`](https://github.com/parse-community/parse-server/blob/8414ad3f3282bc0bb3b43e55396998f4f9a7444d/src/triggers.js#L405) functions.
In both cases, these methods stringify the Parse Object (request & response) before checking for the log level.
As JSON.stringify is one of the most CPU-intensive (and one of the slowest) methods of Parse, I guess checking for log level first would be better.

### PR DESCRIPTION
**Pull request by dplewis**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/parse-server/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/parse-server/blob/alpha/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

## Issue

Currently Cloud Code logs use `JSON.stringify` even if they aren't displayed to the console. `JSON.stringify` is one of the most CPU-intensive and this will improve performance. If you return a large amount of data from a cloud code function `JSON.stringify` will return `Invalid string length` error message.

Closes: https://github.com/parse-community/parse-server/issues/8418

## Approach

Add `silent` support to `logLevels` Parse Server Config. Enable `silent` cloud code logs by default for the test suite to improve speed.

```
{
  ...
  logLevels: {
    cloudFunctionSuccess: 'silent',
    cloudFunctionError: 'silent',
    triggerAfter: 'silent',
    triggerBeforeError: 'silent',
    triggerBeforeSuccess: 'silent',
  },
}
```
## Tasks

- [x] Add tests
- [x] Add changes to documentation (guides, repository pages, code comments)
- [x] Add [security check](https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md#security-checks)
- [x] Add new Parse Error codes to Parse JS SDK

### CODE REVIEW

File: spec/helper.js [hunk 1]
```diff
@@ -444,10 +454,8 @@ global.it_id = (id, func) => {
   if (testExclusionList.includes(id)) {
     return xit;
   } else {
-    if(func === undefined)
-      return it;
-    else
-      return func;
+    if (func === undefined) return it;
+    else return func;
```
Comment (mtrezza) [inline spec/helper.js]:
```suggestion
    return func || it;
```

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — APPROVED** [review]

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.31](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.31)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.1.0-alpha.1](https://github.com/parse-community/parse-server/releases/tag/7.1.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.1.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/7.1.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.1.0](https://github.com/parse-community/parse-server/releases/tag/7.1.0)

**Pull request merged**